src/opnsense/mvc/app/controllers/OPNsense/Base/ControllerBase.php

```diff
@@ -44,32 +44,15 @@ class ControllerBase extends ControllerRoot
      * @param OPNsense\Core\Config $cnf config handle
      * @return ViewTranslator
      */
-    public function getTranslator($cnf)
+    public function getTranslator()
     {
-        $lang = 'en_US';
-
-        foreach ($cnf->object()->system->children() as $key => $node) {
-            if ($key == 'language') {
-                $lang = $node->__toString();
-                break;
-            }
-        }
-
-        $lang_encoding = $lang . '.UTF-8';
-        $textdomain = 'OPNsense';
-
         $ret = new ViewTranslator(array(
             'directory' => '/usr/local/share/locale',
-            'defaultDomain' => $textdomain,
-            'locale' => $lang_encoding,
+            'defaultDomain' => 'OPNsense',
+            'locale' => "en_US.UTF-8",
```

Comment:
I don't think it's a good idea to feed in a static locale here, according to the docs (https://docs.phalconphp.com/en/latest/api/Phalcon_Translate_Adapter_Gettext.html) Gettext() expects a proper one here.